tools/test/audio/eqiir_run.sh

```diff
@@ -12,4 +12,4 @@ fi
 COMP=eq-iir
 DIRECTION=playback
 
-./comp_run.sh $COMP $DIRECTION $1 $2 $3 $3 $4 $5
+./comp_run.sh $COMP $DIRECTION "$1" "$2" "$3" "$3" "$4" "$5"
```

Comment:
Just curious: is shellcheck smart enough not to warn about $COMP and $DIRECTION?

BTW in a future version you may want to use a `main "$@"` like this:
```
main()
{
    local COMP DIRECTION
    COMP=eq-iir
    ...
}
...
main "$@"
```
Always having a `main` even for small scripts means you can use `local`. It also minimizes `diff` output when `main` grows too big and you starting moving code in and out of functions. Also lets you define functions at any place in the script and makes sure the entire script is parsed before it starts running.

Reply:
Yep, no warnings for  $COMP and $DIRECTION there.

I didn't fix this kind of issues because I felt current is better:

```
In tools/test/audio/sof-test-perf.sh line 8:
if [ $? -eq 0 ]; then
     ^-- SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

```

I didn't know about main(). I've earlier placed functions to beginning. Main() allows to start with it I assume so it would be better. Thanks for tip!

Reply:
Main() etc. done